core-blocks/latest-comments/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies.
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import './style.scss';
+import edit from './edit';
+
+export const name = 'core/latest-comments';
+
+export const settings = {
+	title: __( 'Latest Comments' ),
+
+	description: __( 'Show a list of your site’s most recent comments.' ),
+
+	icon: 'list-view',
+
+	category: 'widgets',
+
+	keywords: [ __( 'recent comments' ) ],
+
+	supports: {
+		html: false,
+	},
+
+	getEditWrapperProps( attributes ) {
+		const { align } = attributes;
+
+		// TODO: Use consistent values across the app;
+		// see: https://github.com/WordPress/gutenberg/issues/7908.
+		if ( [ 'left', 'center', 'right', 'wide', 'full' ].includes( align ) ) {
```

Comment:
We should double check that our new ES2015 prototype member polyfills with Babel 7 `useBuiltIns: 'usage'` allows this to work as expected in IE11.

This is an ES2015+ method which until recently was not expected to be polyfilled and would therefore error in IE11.

Reply:
I tested it in IE11 and it worked; I could change the align values and nothing errored on me (it touches this code path).